core/src/main/java/org/apache/accumulo/core/file/rfile/VisMetricsGatherer.java

```diff
@@ -136,12 +136,12 @@ public void printMetrics(boolean hash, String metricWord, PrintStream out) {
         if (hash) {
           String encodedKey = "";
           try {
-            byte[] encodedBytes = MessageDigest.getInstance(Constants.PW_HASH_ALGORITHM)
+            byte[] encodedBytes = MessageDigest.getInstance(Constants.NON_CRYPTO_USE_HASH_ALGORITHM)
```

Comment:
Since this use of the digest is specific to this specific metric utility's serialization, we can probably just use a local constant, rather than one in `Constants.java` (which I'd personally like to phase out over time).